### PR TITLE
disable the number on the announcing board/modal

### PR DESCRIPTION
**Pull request by elBeato**
## Issue: 
#99

Same process is already implemented in the backend.

### CODE REVIEW

**Comment by CarmenUZH** [comment]
Is it possible to add a visual idicator to let the player know that the button is disabled?
Right now they all still look the same but i can't click one of them

**Comment by elBeato** [comment]
Thanx for check my code, indeed there was a bug. Of course the unable amount of score should be disabled.
(Aswell as the code smell)

File: src/stores/CurrentGameStore.ts [hunk 1]
```diff
@@ -89,4 +89,21 @@ export class CurrentGameStore {
...
   }
+
+  //Calculate if only one player, exactlly the last one has to announce the tricks
+  @computed get isLastPlayerAnnouncing() {
+    let nullValues = 0
+    this.activeMatch?.hands.forEach(
+      (el) => el.announcedScore === null ?? (nullValues += nullValues + 1)
+    )
+    if (nullValues === 1) return true
+    else return false
```
Comment (BacLuc) [inline src/stores/CurrentGameStore.ts:100]:
```suggestion
    return this.activeMatch?.hands.filter((el) => el.announcedScore === null).length === 1
```
(Nicht getestet)

Plus ohne anfänger 
if (condition) return true else return false

**Review by BacLuc — APPROVED** [review]

File: src/stores/CurrentGameStore.ts (hunk 1 above)
Comment (elBeato) [reply src/stores/CurrentGameStore.ts:100]: Finde meine Lösung genau so gut...:-)

**Pull request merged**